### DISCUSSION TITLE
Uninstall

### DISCUSSION THREAD

**Discussion by IIIsengard** [ImageMagick/ImageMagick#7564] [Help]
### ImageMagick version

 6.9.13-14 Q16 x86_64 18442 + Cpanel

### Operating system, version and so on

AlmaLinux 8.10

### Description

I used the script below to install imagick as the cpanel version was bad, this is now fixed and I want to remove this version so I can get the proper cpanel version installed.  I am at a loss as how to remove this properly.

cd /usr/src
wget https://codeload.github.com/Imagick/imagick/zip/refs/heads/master
unzip master
rm -fr master
mv imagick-master imagick
cd imagick
/opt/cpanel/ea-php83/root/usr/bin/phpize
./configure --with-php-config=/opt/cpanel/ea-php83/root/usr/bin/php-config
make
make test
make install
echo "extension=imagick.so" >> /opt/cpanel/ea-php83/root/etc/php.d/imagick.ini
service httpd restart
cd ..
rm -fr imagick

**Comment by snibgo** [comment]
This forum is anout ImageMagick, not IMagick. IMagick is a PHP interface to ImageMagick.
